go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/oapi-codegen/runtime v1.1.1
 	github.com/operator-framework/api v0.27.0
 	github.com/operator-framework/operator-lifecycle-manager v0.27.0
-	github.com/percona/everest-operator v0.6.0-dev1.0.20250113063626-b38e7d1b3932
+	github.com/percona/everest-operator v0.6.0-dev1.0.20250126142519-d50affd316d3
```

Comment:
Commenting to remember to update the version before merge after the operator PR is merged